apis/kueue/webhooks/clusterqueue_webhook.go

```diff
@@ -108,6 +111,14 @@ func ValidateClusterQueue(cq *kueue.ClusterQueue) field.ErrorList {
 	return allErrs
 }
 
+// We need to validate the spec.queueingStrategy immutable manually before Kubernetes 1.25.
```

Comment:
Could you create an issue for this? I think we can remove the webhooks in favor of CEL once 1.24 reaches EoL https://kubernetes.io/releases/

Reply:
https://github.com/kubernetes-sigs/kueue/issues/463